.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @KittyCAD/frontend
```

Comment:
How do we want to structure this? We probably need more than just https://github.com/orgs/KittyCAD/teams/frontend here